<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PilotInterface</class>
 <widget class="QMainWindow" name="PilotInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FALCON Pilot Interface - Avionics Style</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #0a0a0a;
    color: #00ff00;
}

QWidget {
    background-color: #0a0a0a;
    color: #00ff00;
}

QLabel {
    color: #00ff00;
    font-weight: bold;
    font-family: &quot;Courier New&quot;, monospace;
}

QPushButton {
    background-color: #1a1a1a;
    color: #00ff00;
    border: 2px solid #00aa00;
    border-radius: 6px;
    padding: 6px;
    font-weight: bold;
    font-family: &quot;Courier New&quot;, monospace;
}

QPushButton:hover {
    background-color: #2a2a2a;
    border-color: #00ff00;
    color: #66ff66;
}

QPushButton:pressed {
    background-color: #0d0d0d;
    border-color: #00ff00;
}

QPushButton:checked {
    background-color: #003300;
    border-color: #00ff00;
    color: #ffffff;
}

QTextEdit {
    background-color: #0d0d0d;
    color: #00ff00;
    border: 2px solid #00aa00;
    border-radius: 4px;
    padding: 4px;
    font-family: &quot;Courier New&quot;, monospace;
    selection-background-color: #004400;
}

QProgressBar {
    border: 2px solid #00aa00;
    border-radius: 4px;
    text-align: center;
    background-color: #1a1a1a;
    color: #00ff00;
    font-family: &quot;Courier New&quot;, monospace;
}

QProgressBar::chunk {
    background-color: #00ff00;
    border-radius: 2px;
}

QSlider::groove:vertical {
    background: #1a1a1a;
    width: 10px;
    border-radius: 5px;
    border: 1px solid #00aa00;
}

QSlider::handle:vertical {
    background: #00ff00;
    border: 2px solid #00aa00;
    height: 24px;
    border-radius: 12px;
    margin: 0 -7px;
}

QSlider::sub-page:vertical {
    background: #1a1a1a;
    border-radius: 5px;
}

QSlider::add-page:vertical {
    background: #00aa00;
    border-radius: 5px;
}

/* 특별한 상태 스타일 */
.status-ready {
    background-color: #001a00;
    border: 2px solid #00ff00;
    color: #00ff00;
}

.status-listening {
    background-color: #001a00;
    border: 2px solid #00ff00;
    color: #00ff00;
}

.status-processing {
    background-color: #001a00;
    border: 2px solid #00ff00;
    color: #00ff00;
}

.status-speaking {
    background-color: #001a00;
    border: 2px solid #00ff00;
    color: #00ff00;
}

/* 경고 상태 */
.alert-high {
    background-color: #003300;
    border: 2px solid #00ff00;
    color: #00ff00;
}

.alert-cleared {
    background-color: #003300;
    border: 2px solid #00ff00;
    color: #00ff00;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_mic">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>161</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #001a00;
    border: 4px solid #00ff00;
    border-radius: 6px;
    color: #00ff00;
}
QPushButton:hover {
    background-color: #003300;
    border-color: #66ff66;
}
QPushButton:pressed {
    background-color: #000d00;
}</string>
    </property>
    <property name="text">
     <string>MIC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rwy_a">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>115</y>
      <width>141</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #00ff00;
    background-color: #1a1a1a;
    border: 2px solid #00aa00;
    border-radius: 4px;
    padding: 4px;
}</string>
    </property>
    <property name="text">
     <string>RWY A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_rwy_a">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>115</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #001a33;
    border: 2px solid #66ccff;
    color: #66ccff;
}
QPushButton:hover {
    background-color: #002244;
    border-color: #88ddff;
}
QPushButton:pressed {
    background-color: #001122;
}</string>
    </property>
    <property name="text">
     <string>CLEARED</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_rwy_b">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>185</y>
      <width>141</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #00ff00;
    background-color: #1a1a1a;
    border: 2px solid #00aa00;
    border-radius: 4px;
    padding: 4px;
}</string>
    </property>
    <property name="text">
     <string>RWY B</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_rwy_b">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>185</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #001a33;
    border: 2px solid #66ccff;
    color: #66ccff;
}
QPushButton:hover {
    background-color: #002244;
    border-color: #88ddff;
}
QPushButton:pressed {
    background-color: #001122;
}</string>
    </property>
    <property name="text">
     <string>CLEARED</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_bird_alert">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>255</y>
      <width>141</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #00ff00;
    background-color: #1a1a1a;
    border: 2px solid #00aa00;
    border-radius: 4px;
    padding: 4px;
}</string>
    </property>
    <property name="text">
     <string>BIRD ALERT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_bird">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>255</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #001a33;
    border: 2px solid #66ccff;
    color: #66ccff;
}
QPushButton:hover {
    background-color: #002244;
    border-color: #88ddff;
}
QPushButton:pressed {
    background-color: #001122;
}</string>
    </property>
    <property name="text">
     <string>CLEAR</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_tts_volume">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #00ff00;
}</string>
    </property>
    <property name="text">
     <string>SPK VOLUME</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="slider_tts_volume">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>130</y>
      <width>30</width>
      <height>171</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>70</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_tts_value">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>190</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #00ff00;
}</string>
    </property>
    <property name="text">
     <string>70</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_stt_volume">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>90</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #00ff00;
}</string>
    </property>
    <property name="text">
     <string>MIC VOLUME</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_stt_volume">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>130</y>
      <width>30</width>
      <height>171</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_stt_result">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #00ff00;
}</string>
    </property>
    <property name="text">
     <string>STT RESULT:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_stt_result">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>400</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Speech recognition result will be displayed here...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tts_text">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>340</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #00ff00;
}</string>
    </property>
    <property name="text">
     <string>TTS RESPONSE:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_tts_input">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>370</y>
      <width>400</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>TTS response text will be displayed here...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-color: #001a00;
    border: 2px solid #00ff00;
    border-radius: 8px;
    padding: 8px;
    color: #00ff00;
}</string>
    </property>
    <property name="text">
     <string>READY</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_utc_time">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #00ff00;
}</string>
    </property>
    <property name="text">
     <string>UTC: 00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_local_time">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #00ff00;
}</string>
    </property>
    <property name="text">
     <string>LOCAL: 00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_callsign">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #00ff00;
}</string>
    </property>
    <property name="text">
     <string>CALLSIGN: KAL123</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_system_title">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>29</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #00ff00;
    background-color: #0d0d0d;
    border: 3px solid #00aa00;
    border-radius: 8px;
    padding: 8px;
}</string>
    </property>
    <property name="text">
     <string>FALCON AVIONICS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
